pkgs/development/compilers/scryer-prolog/default.nix

```diff
@@ -13,30 +13,21 @@
 
 rustPlatform.buildRustPackage rec {
   pname = "scryer-prolog";
-  version = "0.9.0";
+  version = "0.9.1";
 
   src = fetchFromGitHub {
     owner = "mthom";
     repo = "scryer-prolog";
     rev = "v${version}";
-    sha256 = "3NHpEg6QaUaqbBCq8uM5hFcqS24q4XrOnKjMmn8Z1Dg=";
+    sha256 = "bDLVOXX9nv6Guu5czRFkviJf7dBiaqt5O8SLUJlcBZo=";
   };
 
   cargoPatches = [
     # Use system openssl, gmp, mpc and mpfr.
     ./cargo.patch
-
-    ./fix-tests.patch
-
-    # Avoid testing failing with "couldn't save history"
-    (fetchpatch {
```

Comment:
Please remove the unused fetchpatch from inputs.

Reply:
Done, thanks for catching that.